src/main/java/com/researchspace/dataverse/api/v1/DatasetOperations.java

```diff
@@ -69,6 +70,15 @@ public interface DatasetOperations {
 	 */
 	void uploadFile(String doi, File file);
 
+	/**
+	 * Uploads a file using a data stream.
+	 *
+	 * @param doi Identifier of the dataset that we are attacking.
```

Comment:
I don't think 'attacking' is a good word here
```suggestion
	 * @param doi Identifier of the dataset that we are sending the data too
```